R/progress.R

```diff
@@ -46,8 +46,10 @@ progress_bar <- function(type, con) {
         first <<- FALSE
       }
       cat("\rDownloading: ", bytes(now, digits = 2), "     ", sep = "", file = con)
-      if (now == total) cat("\n", file = con)
```

Comment:
This code needs to remain as is - this is for the case where curl doesn't know the total download size. 

I think you should add this code below `setTxtProgressBar` so two branches of the if are parallel.